percona-packaging/redhat/percona-server-mongodb.spec.template

```diff
@@ -1,13 +1,13 @@
 #
 %define         mongo_home /var/lib/mongo
 #
-Name:           Percona-Server-MongoDB-36
+Name:           Percona-Server-MongoDB
 Version:        @@VERSION@@
 Release:        @@RELEASE@@%{?dist}
 Summary:        High-performance MongoDB replacement from Percona (metapackage)
 Group:          Applications/Databases
 
-License:        AGPL
+License:        AGPL 3.0
```

Comment:
upstream uses SSPL. Are you sure we still on AGPL?

Reply:
According to https://github.com/mongodb/mongo/blob/v4.0/rpm/mongodb-org-unstable-init.spec#L7
it is still AGPL